rhizome/lib/vm_setup.rb

```diff
@@ -92,7 +92,8 @@ def subdivide_network(net)
     [halved, halved.next_sib]
   end
 
-  def routes(gua, private_subnets)
+  def routes(gua, private_subnets, ndp_needed)
+    main_device = r "ip route | awk '/default/ {print $5}'"
```

Comment:
This is admirably terse, but I wonder if it is as sound as `ip -j route` would be. Is the latter _also_ short, or short-ish? It would be a surplus in soundness if we could do it that way.

Reply:
here you go